packages/lucide/src/lucide.ts

```diff
@@ -5,7 +5,7 @@ import * as allIcons from './icons';
  * Replaces all elements with matching nameAttr with the defined icons
  * @param {{ icons?: object, nameAttr?: string, attrs?: object }} options
  */
-const createIcons = ({ icons = {}, nameAttr = 'icon-name', attrs = {} } = {}) => {
```

Comment:
We maybe should consider that we have an overlapping time, with changing this.
So maybe we should support the old attribute as well and console a warning, with saying we remove this in v1?

Reply:
Yeah, good idea, I've posted an implementation of this, also checking if the default attribute name was used.